--- hfst-3.14.0/back-ends/foma/Makefile.am~	2017-05-22 14:58:33.000000000 +0200
+++ hfst-3.14.0/back-ends/foma/Makefile.am	2018-07-06 09:22:44.784803183 +0200
@@ -15,7 +15,7 @@ libfoma_la_SOURCES = int_stack.c define.
         coaccessible.c io.c utf8.c spelling.c dynarray.c mem.c stringhash.c \
         trie.c lex.lexc.c lex.yy.c regex.c
 
-hfst_foma_LDADD = $(top_builddir)/back-ends/foma/libfoma.la
+hfst_foma_LDADD = libfoma.la
 
 if WANT_READLINE
 hfst_foma_LDFLAGS=-lreadline -lz
