--- hfst-3.4.5/configure.ac.orig	2013-06-10 12:58:12.000000000 +0200
+++ hfst-3.4.5/configure.ac	2013-06-28 19:41:03.140615707 +0200
@@ -533,7 +533,8 @@
 
 # Checks for libraries
 
-AC_CHECK_LIB([xml2], [main])
+AC_CHECK_LIB([xml2], [main], [LIBXML2_LIBS="-lxml2"], [LIBXML2_LIBS=])
+AC_SUBST([LIBXML2_LIBS])
 AC_LANG_PUSH([C++])
 AS_IF([test "x$with_openfst" != "xno" -a "x$enable_mingw" == "xno"], 
       [AC_CHECK_LIB([dl], [main])])
