--- hfst-2.4.1/hfst-tools/hfst-lexc/src/Makefile.am.orig	2009-11-10 23:30:20.000000000 +0100
+++ hfst-2.4.1/hfst-tools/hfst-lexc/src/Makefile.am	2010-11-27 12:50:55.227094909 +0100
@@ -1,9 +1,8 @@
 ## Process this file with automake to produce Makefile.in
 
 ## For nested distribution as part of hfst
-hfst_lexc_LDADD = ../../../hfst2/libhfst.la libhfstlexc.la
+hfst_lexc_LDADD = libhfstlexc.la
 AM_CPPFLAGS = -I../../../ -I ../../../hfst2
-AM_CXXFLAGS = -O3
 
 lib_LTLIBRARIES=libhfstlexc.la
 bin_PROGRAMS=hfst-lexc
@@ -20,6 +19,8 @@
 libhfstlexc_la_SOURCES=\
 					   lexc.cc string-munging.cc lexcio.cc \
 					   xducer.cc xymbol.cc xymbol-table.cc xymbol-bridges.cc
+libhfstlexc_la_LIBADD=../../../hfst2/libhfst.la
+
 BUILT_SOURCES=lexc-parser.h
 
 hfst_lexc_CXXFLAGS=-Wno-deprecated
