--- hfst-3.2.0/libhfst/hfst.pc.in.orig	2011-07-30 15:37:58.000000000 +0200
+++ hfst-3.2.0/libhfst/hfst.pc.in	2011-08-02 19:24:49.316577950 +0200
@@ -7,4 +7,4 @@
 Description: Finite-state transducer library bridge for multiple FLOSS packages
 Version: @LIBHFST_VERSION@
 Libs: -L${libdir} -l@LIBHFST_NAME@ @LIBS@
-Cflags: -I${includedir}/hfst/ @GLIB_CFLAGS@
+Cflags: -I${includedir}/hfst @GLIB_CFLAGS@
